### PR TITLE
build leaderboard using localStorag

### PR DESCRIPTION
**Pull request by boo-shehab**
# Hi reviewer 👋 
## Description of the task to be completed
 - create the branch according to the git flow
 - create the npm webpack setup
 - create the index.html index.css index.js files
 - create the function to add a list and display a list
 - check the linters

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    </div>
+  </header>
+  <section>
+    <div class="container">
+      <div class="leader-board">
+        <div class="refresh">
+          <span>Refresh the page</span>
+          <button>refersh</button>
+        </div>
+        <div class="board">
```
Comment (thecodechaser) [inline src/index.html:21]: - [ ] Please make sure the refresh text and the button is outside of the box as on the wireframe.

### Your design
image

### Wireframe design
image

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <h2>Add your score</h2>
+        <form action="">
+          <label for="name" hidden> add your name</label>
+          <input type="text" id="name" placeholder="name">
+          <label for="score" hidden> add your score</label>
+          <input type="text" id="score" placeholder="score">
+          <button type="submit">submit</button>
+        </form>
+      </div>
+    </div>
```
Comment (thecodechaser) [inline src/index.html:60]: - [ ] Please make sure there's no top margin in this section and the submit button should align to the right side of the screen to match the wireframe design.

### Your design
image

### Wireframe design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @boo-shehab👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+section .add-score {
+  text-align: center;
+}
+
+section .add-score form {
+  display: grid;
+  justify-items: right;
+  gap: 20px;
+}
```
Comment (Gopxfs) [inline src/style.css:95]: As requested by the previous reviewer, please make sure to align your sections 😉:
| Wireframe  | Your design |
| ------------- | ------------- |
| image  | image |

Kindly notice that, in the original design, both sections are aligned, while in your design, the section on the right is centered.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @boo-shehab 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Implemented some of the changes requested by the previous reviewer.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimalamiri — APPROVED** [review]
Hi @boo-shehab, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- The layout is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right Git Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**